Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include "Person.h"
+
+Person::Person(){
```

Comment:
We typically like { on a new line.

Ex.
```
Person::Person()
{
  //code here
}
```

It's not your fault though because nothing in our [style guide](https://github.com/UCSolarCarTeam/Development-Resources/tree/master/Development-Guide/Style-Guide) tells you about this.